### PR TITLE
Guess terminal to use as default app when using scripts/xephyr

### PR DESCRIPTION
**Pull request by SirWrexes**

### CODE REVIEW

File: scripts/guess_terminal [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+mate-terminal
+kitty
+yakuake
+tilda
+guake
+eterm
+st
+urxvt
+xterm
+x-terminal-emulator"
```
Comment (tych0) [inline scripts/guess_terminal:22]: Can we make this a python script that uses our existing guess_terminal() instead of copy/pasting the list?

**Comment by m-col** [comment]
Nice, great idea.

**Comment by m-col** [comment]
You could also just stick this where you have `$(./guess_terminal)`:
```
$(python -c "from libqtile.utils import guess_terminal; print(guess_terminal()))"
```

**Comment by SirWrexes** [comment]
Huh. I guess I should have thought about that.
This might as well just be in the script rather than a separate file since it fits in one line. 🤔

**Comment by m-col** [comment]
Please could you squash the commits down into one? Otherwise looks good.

**Comment by SirWrexes** [comment]
Oh I'm sorry I forgot to remove the guess_terminal script. Hold on.

**Comment by m-col** [comment]
Awesome, thanks for this!

**Pull request merged**